src/current/_includes/v23.1/misc/tooling.md

```diff
@@ -4,7 +4,16 @@ Cockroach Labs has partnered with open-source projects, vendors, and individuals
 
 - **Full support** indicates that Cockroach Labs is committed to maintaining compatibility with the vast majority of the tool's features. CockroachDB is regularly tested against the latest version documented in the table below.
 - **Beta support** indicates that Cockroach Labs is working towards full support for the tool. The primary features of the tool are compatible with CockroachDB (e.g., connecting and basic database operations), but full integration may require additional steps, lack support for all features, or exhibit unexpected behavior.
-- **Partner supported** indicates that Cockroach Labs has a partnership with a third-party vendor that provides support for the CockroachDB integration with their tool. 
+- **Partner supported** indicates that Cockroach Labs has a partnership with a third-party vendor that provides support for the CockroachDB integration with their tool.
+
+{{site.data.alerts.callout_danger}}
+Tools, drivers, or frameworks are considered **unsupported** if:
+
+- The tool, driver, or framework is not listed on this page.
+- The version of a supported tool, driver, or framework is not listed on this page.
+
+Cockroach Labs support for unsupported tools, drivers, and frameworks is considered "best effort," meaning we won't be able to fully help you troubleshoot errors in your deployment.
+{{site.data.alerts.end}}
```

Comment:
@dikshant: would it also be worth adding some warning that around moving production workloads to CockroachDB using unsupported drivers?

Reply:
Yeah, for paying customers we should suggest they evaluate this with their account team before considering to do this.

Reply:
We should also add a bit about raising unsupported tooling related issues with the tool maintainers directly.